kectl/app/cmd/cloud.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
These pages looks like a duplicate from pr #435. Please delete from 1 place.
The PR description says that this PR is only for edge commands so I guess these pages should be deleted from this PR
cc @srivatsav123